### PR TITLE
Version Packages

### PR DESCRIPTION
**Pull request by hashibot-web**
This PR was opened by the [Changesets release](https://github.com/changesets/action) GitHub action. When you're ready to do a release, you can merge this and the packages will be published to npm automatically. If you're not ready to do a release yet, that's fine, whenever you add more changesets to main, this PR will be updated.

# Releases
### Minor Changes

-   [#158](https://github.com/hashicorp/web-platform-packages/pull/158) [`ef2b0f5`](https://github.com/hashicorp/web-platform-packages/commit/ef2b0f5b303b7752ae95ae217ea887119462c9fa) Thanks [@BRKalow](https://github.com/BRKalow)! - Adds a new docs section on validating content with our content checks.

 ## @hashicorp/integrations-hcl@0.2.0

### Minor Changes

-   [#161](https://github.com/hashicorp/web-platform-packages/pull/161) [`22c873c`](https://github.com/hashicorp/web-platform-packages/commit/22c873cc2fde5eaa5ad5d2577512a597f4b89333) Thanks [@thiskevinwang](https://github.com/thiskevinwang)! - ### Breaking

    This updates the interface to require 'organization_id' as well as updates the internal API Client.

 ## @hashicorp/platform-content-conformance@0.0.7

### Patch Changes

-   [#159](https://github.com/hashicorp/web-platform-packages/pull/159) [`d99438d`](https://github.com/hashicorp/web-platform-packages/commit/d99438dccbede772bf3bccfd08bd141d4278bfac) Thanks [@BRKalow](https://github.com/BRKalow)! - Adjust CLI output, improve erorr message for frontmatter rule"

### CODE REVIEW

File: packages/integrations-hcl/CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,15 @@
 # @hashicorp/integrations-hcl
 
-## 0.1.0
+## 0.2.0
+
 ### Minor Changes
 
+- [#161](https://github.com/hashicorp/web-platform-packages/pull/161) [`22c873c`](https://github.com/hashicorp/web-platform-packages/commit/22c873cc2fde5eaa5ad5d2577512a597f4b89333) Thanks [@thiskevinwang](https://github.com/thiskevinwang)! - ### Breaking
+
+  This updates the interface to require 'organization_id' as well as updates the internal API Client.
```
Comment (thiskevinwang) [inline packages/integrations-hcl/CHANGELOG.md:9]: @BRKalow Are we allow to update these generated changelogs? I just realized this text is slightly incorrect

Comment (thiskevinwang) [inline packages/integrations-hcl/CHANGELOG.md:9]:
```suggestion
  This change expects a new format for the 'identifier' param. This also updates the internal API Client which expects API changes in https://github.com/hashicorp/integrations-api/pull/79.
```

Comment (BRKalow) [reply packages/integrations-hcl/CHANGELOG.md:9]: I _think_ so 💭

Comment (thiskevinwang) [reply packages/integrations-hcl/CHANGELOG.md:9]: Doin' it 😐

**Review by thiskevinwang — APPROVED** [review]
👍

File: packages/integrations-hcl/CHANGELOG.md (hunk 1 above)
Comment (thiskevinwang) [reply packages/integrations-hcl/CHANGELOG.md:9]: My manual change got force-pushed over

**Pull request merged**